avocado_vt/vt_agent/src/avocado_vt/agent/services/core.py

```
@@ -0,0 +1,534 @@
+        suite_data = {}
+        if os.path.exists(state_file):
+            with open(state_file, "r") as f:
+                suite_data["state"] = json.load(f)
+        if os.path.exists(result_file):
+            with open(result_file, "r") as f:
+                suite_data["results"] = json.load(f)
+
+        LOG.info(
+            "Successfully loaded suite data for job '%s', suite '%s'", job_id, suite_id
+        )
+        return suite_data if suite_data else None
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)